terraform/groups/ecs-service/main.tf

```diff
@@ -30,25 +30,25 @@ module "ecs-service" {
   task_execution_role_arn = data.aws_iam_role.ecs_cluster_iam_role.arn
 
   # Load balancer configuration
-  lb_listener_arn                 = data.aws_lb_listener.service_lb_listener.arn
-  lb_listener_rule_priority       = local.lb_listener_rule_priority
-  lb_listener_paths               = local.lb_listener_paths
-  multilb_setup                   = true
-  multilb_listeners               = {
-    "priv-api-lb": {
-          listener_arn            = data.aws_lb_listener.secondary_lb_listener.arn,
-          load_balancer_arn       = data.aws_lb.secondary_lb.arn
-      }
-    "pub-api-lb": {
-      load_balancer_arn           = data.aws_lb.service_lb.arn
-      listener_arn                = data.aws_lb_listener.service_lb_listener.arn
-    }      
+  lb_listener_arn           = data.aws_lb_listener.service_lb_listener.arn
+  lb_listener_rule_priority = local.lb_listener_rule_priority
+  lb_listener_paths         = local.lb_listener_paths
+  multilb_setup             = true
+  multilb_listeners = {
+    "priv-api-lb" : {
+      listener_arn      = data.aws_lb_listener.secondary_lb_listener.arn,
+      load_balancer_arn = data.aws_lb.secondary_lb.arn
+    }
+    "pub-api-lb" : {
+      load_balancer_arn = data.aws_lb.service_lb.arn
+      listener_arn      = data.aws_lb_listener.service_lb_listener.arn
+    }
   }
 
   # ECS Task container health check
   use_task_container_healthcheck = true
-  healthcheck_path          = local.healthcheck_path
-  healthcheck_matcher       = local.healthcheck_matcher
+  healthcheck_path               = local.healthcheck_path
+  healthcheck_matcher            = local.healthcheck_matcher
```

Comment:
Do wee need to add min_task_count                     = var.min_task_count on line 65+/-?